Sources/MapboxMaps/Foundation/Extensions/Core/OfflineRegionGeometryDefinition.swift

```diff
@@ -0,0 +1,15 @@
+@_implementationOnly import MapboxCoreMaps_Private
+import MapboxCommon
+
+extension MapboxCoreMaps.OfflineRegionGeometryDefinition {
+
+    convenience init(styleURL: String, geometry: Turf.Geometry, minZoom: Double, maxZoom: Double, pixelRatio: Float, glyphsRasterizationMode: GlyphsRasterizationMode) {
```

Comment:
should be public

Reply:
and needs a documentation comment - I recommend copying info from `MBMOfflineRegionGeometryDefinition.h`